result/cargo/osv_scanner.txt

```diff
@@ -0,0 +1,13 @@
+Scanning dir ./docker/cargo/
+Scanned /usr/local/google/home/gongh/Desktop/code/DockerTester/docker/cargo/example/Cargo.lock file and found 34 packages
+Scanning directory for vendored libs: /usr/local/google/home/gongh/Desktop/code/DockerTester/docker/cargo/example/target/debug/deps
+Scanning directory for vendored libs: /usr/local/google/home/gongh/Desktop/code/DockerTester/docker/cargo/example/target/release/deps
+Begin building rust/cargo project
```

Comment:
I am not sure if all vulns below were called (no "uncalled vulns" entries but h2 package was not executed) even I passed "--call-analysis=all" to the command.